compatibility-verifier/compCheck.sh

```diff
@@ -70,21 +70,31 @@ function waitForClusterReady() {
   sleep 2
   echo "Cluster ready."
 }
+
+#set config file is present or not
+function setConfigFile() {
+  if [[ -n $1 ]]; then
```

Comment:
why don't we just check if file exists (using -f) here and populate the configFileArg?